DNN Platform/Website/DotNetNuke.Website.csproj

```diff
@@ -0,0 +1,6882 @@
+    <Content Include="Resources\Shared\components\CodeEditor\mode\scheme\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\scheme\scheme.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\shell\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\shell\shell.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\shell\test.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\sieve\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\sieve\sieve.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\slim\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\slim\slim.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\slim\test.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\smalltalk\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\smalltalk\smalltalk.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\smartymixed\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\smartymixed\smartymixed.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\smarty\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\smarty\smarty.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\solr\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\solr\solr.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\soy\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\soy\soy.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\sparql\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\sparql\sparql.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\spreadsheet\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\spreadsheet\spreadsheet.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\sql\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\sql\sql.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stex\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stex\stex.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stex\test.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stylus\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\stylus\stylus.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\swift\index.html" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\swift\swift.js" />
+    <Content Include="Resources\Shared\components\CodeEditor\mode\tcl\index.html" />
+    
```

Comment:
Looks like this file got duplicated in the merge

Reply:
Yep, just resubmitted, let's see...